### PR TITLE
Feature/sdl passenger mode 0119

### LINKED ISSUE #134: [SDL 0119] SDL Passenger Mode
Proposal: [SDL Passenger Mode](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0119-SDL-passenger-mode.md)

> Allow a passenger to dismiss the lock screen to improve usability of applications. This re-visited proposal now includes a way for individual OEMs to control this mode via policy updates.

Review: https://github.com/smartdevicelink/sdl_evolution/issues/359

Steering Committee Decision:

>The Steering Committee voted to accept this proposal with the following revisions:

- lockScreenDismissalEnabled param should not be mandatory
- Swipe method to dismiss the lock screen
- Be able to re-engage the lock screen if a notification is received, instead of allowing a lock screen to be disabled for the remainder of an app's session

>The proposal .md was updated on April 12, 2018 to include the agreed upon revisions.
>The proposal .md was updated again on June 26, 2019 to include revisions as outlined in accepted Evolution Proposal [SDL 0119 Revisions - SDL-passenger-mode](https://github.com/smartdevicelink/sdl_evolution/issues/769), which included expanding the feature to impact the Policy Server.

### PR DESCRIPTION
**Pull request by russjohnson09**
Fixes #134 

This PR is **ready** for review.

### Risk
This PR makes **minor** API changes.

### Testing Plan
Mocha tests are already in place for updating the module config. These tests still pass and an additional test was added to ensure that lock_screen_dismissal_enabled is set properly.

### Summary
1. Added a new default warning message in english only for the lock screen dismissal.
2. Added the lock_screen_dismissal_enabled to the module_config table, in the frontend, and as the response in the appropriate api calls.

##### Breaking Changes
None

##### Enhancements
*  Added the lock_screen_dismissal_enabled option.

##### Bug Fixes
None

### Tasks Remaining:
None

### CLA
- [x] I have signed [the CLA](https://docs.google.com/forms/d/e/1FAIpQLSdsgJY33VByaX482zHzi-xUm49JNnmuJOyAM6uegPQ2LXYVfA/viewform)

### CODE REVIEW

File: test/api/v1/module/promote.js [hunk 1]
```diff
@@ -33,6 +33,93 @@ common.post(
     }
 );
 
+
```
Comment (nickschwab) [inline test/api/v1/module/promote.js]: Please remove these granular unit tests in favor of adding `lock_screen_dismissal_enabled` to the existing tests.

File: test/api/v1/module/promote.js [hunk 2]
```diff
@@ -33,6 +33,93 @@ common.post(
...
+common.get(
```
Comment (nickschwab) [inline test/api/v1/module/promote.js]: Please remove these granular unit tests in favor of adding lock_screen_dismissal_enabled to the existing tests.

Comment (russjohnson09) [reply test/api/v1/module/promote.js]: @nickschwab I've updated the test changes to be more in line with existing tests.

**Comment by nickschwab** [comment]
Approved! Hold for merge in the future.

**Pull request merged**